src/components/buttons/MiniButton.js

```diff
@@ -56,9 +50,9 @@ export default function MiniButton({
           shadows={disabled ? shadows.disabled : shadows.default}
           width={width}
         />
-        <Content>
+        <Content hasLeadingIcon={hasLeadingIcon}>
           {typeof children === 'string' ? (
-            <Text color="white" weight="semibold">
+            <Text align="center" color="white" weight="bold">
```

Comment:
```suggestion
            <Text align="center" color="white" weight={fonts.weight.semibold}>
```

Reply:
@osdnk is this necessary?

think our text component handles font weights

cc @mikedemarais

Reply:
hey! yes our `<Text />` component accepts all the keys of our `fonts.weights` as props

i know that we are pretty inconsistent about this using this API, and other style/layout APIs i’ve made — i definitely think the APIs can be improved and that we can consolidate patterns

but i agree with @christianbaroni that it’s simpler to just use “bold” here 

very down to discuss / flesh out better style APIs

cc @osdnk 

also i notice that michał’s suggestion not only changes the syntax but also from “bold” to “semibold” — not sure if the weight change was intentional or not (cc @christianbaroni) or if he was just intending to suggest syntax change 

😘